### ISSUE TITLE
I have one more question about "train objective" and "val objective" in result after training by running "train_SRnet.m"

### ISSUE THREAD

**Issue by star4s** [layumi/2016_super_resolution#13]
I have one more question about "train objective" and "val objective" in result after training by running "train_SRnet.m"

After I take training by running "train_SRnet.m", I got a bottom graph image.

![1](https://cloud.githubusercontent.com/assets/25783167/26228058/53313780-3c71-11e7-8b9b-9e1f29322a61.jpg)

What meaning is the gap between "train objective" and "val objective"?
The "train objective" blue line is bottom against the "val objective" red line over epoch 3.
I am so sorry every time, I am bother you.
Thank you very much for your attention.

**Comment by layumi** [comment]
1. Train: CNN use the training data to tune the parameters so it usually over-fits the training set. (show a lower loss value)
2. Validation: CNN did not see the validation data before. The validation result can reflect the "real" situation in the test setting. So it usually achieves a little bit higher loss.
